docusaurus.config.ts

```diff
@@ -233,11 +233,11 @@ const config: Config = {
       theme: prismThemes.github,
       darkTheme: prismThemes.dracula,
     },
-    algolia: {
```

Comment:
Remove unused codes?

Reply:
Can keep it like this. Maybe will reuse algolia for the website search in the future.